classes/shop/Shop.php

```diff
@@ -397,7 +397,7 @@ public static function initialize()
         );
 
         $isAllShop = 'all' === $id_shop;
-        $isApiInUse = defined('_PS_API_IN_USE_') && _PS_API_IN_USE_;
+        $isApiInUse = defined('_PS_API_IN_USE_') && _PS_API_IN_USE_ === true;
```

Comment:
I'm not sure why this was needed since it's only use for Webservice context and the only value it can have so far `true` anyway https://github.com/PrestaShop/PrestaShop/blob/31bdccd81b2204e2f823ba3b5fa7590924e4e644/webservice/dispatcher.php#L32

Is it really in your PR's scope?

Reply:
oh, no I fixed it in another one. Forgot here, thanks :+1: